database/rrd.h

```diff
@@ -164,6 +164,7 @@ struct label {
 };
 
 struct label *create_label(char *key, char *value, LABEL_SOURCE label_source);
+extern int is_valid_label(char *key);
```

Comment:
we'll only use this in rrdhost.c, I don't think you need an extern here

Reply:
All right, I was in doubt about the possibility to call it out, thank you! I removed it from rrd.h and I used static in the file that it is declared.